### PR TITLE
Updated Encoding parameter and YAML for Out-File

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Related to Issue #2000
Updated Encoding parameter for Out-File
Style updates
Paragraph reflows
Fixed links

Acrolinx scores
Original version: 92
Updated version: 96

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[db8d607](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/db8d607a4aa7c1e08b22914831c096cca95629e2)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[db8d607](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/db8d607a4aa7c1e08b22914831c096cca95629e2)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[db8d607](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/db8d607a4aa7c1e08b22914831c096cca95629e2)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/3.0/Microsoft.PowerShell.Utility/Out-File.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-3.0&branch=pr-en-us-3666) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FOut-File.md)
[reference/4.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/4.0/Microsoft.PowerShell.Utility/Out-File.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-4.0&branch=pr-en-us-3666) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Utility%2FOut-File.md)
[reference/5.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.0/Microsoft.PowerShell.Utility/Out-File.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-5.0&branch=pr-en-us-3666) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FOut-File.md)
[reference/5.1/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.1/Microsoft.PowerShell.Utility/Out-File.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-5.1&branch=pr-en-us-3666) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FOut-File.md)
[reference/6/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/6/Microsoft.PowerShell.Utility/Out-File.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-6&branch=pr-en-us-3666) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Utility%2FOut-File.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FOut-File.md"></a>
### [reference/3.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/3.0/Microsoft.PowerShell.Utility/Out-File.md)
 - **[Warning]** Invalid file link:(~/3.0/Microsoft.Powershell.Core/About/about_Providers).

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Utility%2FOut-File.md"></a>
### [reference/4.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/4.0/Microsoft.PowerShell.Utility/Out-File.md)
 - **[Warning]** Invalid file link:(~/4.0/Microsoft.Powershell.Core/About/about_Providers).

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FOut-File.md"></a>
### [reference/5.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.0/Microsoft.PowerShell.Utility/Out-File.md)
 - **[Warning]** Invalid file link:(~/5.0/Microsoft.Powershell.Core/About/about_Providers).

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FOut-File.md"></a>
### [reference/5.1/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.1/Microsoft.PowerShell.Utility/Out-File.md)
 - **[Warning]** Invalid file link:(~/5.1/Microsoft.Powershell.Core/About/about_Providers).

<a name="reference%2F6%2FMicrosoft.PowerShell.Utility%2FOut-File.md"></a>
### [reference/6/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/6/Microsoft.PowerShell.Utility/Out-File.md)
 - **[Warning]** Invalid file link:(~/6/Microsoft.Powershell.Core/About/about_Providers).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C7%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902070204312135-3666%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2QQIt%2B0v%2BdTdXMi9Bm1p7V6CI4HTVRa0iK5RIHi3YM4%3D&st=2019-02-07T02%3A13%3A38Z&se=2019-03-10T02%3A18%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[71f996d](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/71f996d13e517e7e9dbb0cd65443e3bd7e249a1b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by davidsmatlak** [comment]
Fixed broken links for about_Providers.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[71f996d](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/71f996d13e517e7e9dbb0cd65443e3bd7e249a1b)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[71f996d](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/71f996d13e517e7e9dbb0cd65443e3bd7e249a1b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/3.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-3.0&branch=pr-en-us-3666) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/4.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-4.0&branch=pr-en-us-3666) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-5.0&branch=pr-en-us-3666) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.1/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-5.1&branch=pr-en-us-3666) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/6/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-6&branch=pr-en-us-3666) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C7%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902070247120552-3666%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o9cesk%2B6SG8ijcHtBc4TXTwONWLAuscL4Ol9Stb19g0%3D&st=2019-02-07T02%3A55%3A41Z&se=2019-03-10T03%3A00%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Out-File.md [hunk 1]
```diff
@@ -1,108 +1,141 @@
...
+Out-File [[-Encoding] <string>] -LiteralPath <string> [-Append] [-Force] [-NoClobber] [-Width <int>]
+[-InputObject <psobject>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-The `Out-File` cmdlet sends output to a file.
-You can use this cmdlet instead of the redirection operator (\>) when you need to use its parameters.
+The `Out-File` cmdlet sends output to a file. When you need to specify parameters for the output use
+`Out-File` rather than the redirection operator (`\>`).
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Out-File.md]: You don't need the backslash when you put it a code fence. So the redirection operator should be (\`>\`). Fix this in all versions.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d029d6b](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/d029d6b51c3cd126446b7a6e8ee649b3459c5ab5)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

File: reference/3.0/Microsoft.PowerShell.Utility/Out-File.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Utility/Out-File.md]: Thanks. I made the change in all versions and updated the PR.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d029d6b](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/d029d6b51c3cd126446b7a6e8ee649b3459c5ab5)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[d029d6b](https://github.com/davidsmatlak/PowerShell-Docs/pull/3666/commits/d029d6b51c3cd126446b7a6e8ee649b3459c5ab5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/3.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-3.0&branch=pr-en-us-3666) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/4.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-4.0&branch=pr-en-us-3666) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.0/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-5.0&branch=pr-en-us-3666) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/5.1/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-5.1&branch=pr-en-us-3666) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Out-File.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-outfile/reference/6/Microsoft.PowerShell.Utility/Out-File.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/out-file?view=powershell-6&branch=pr-en-us-3666) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C7%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902071705136006-3666%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dMgTrg%2FBihCPuRron%2BP30VVhveIS9l%2BAXgUUPgsXhhw%3D&st=2019-02-07T17%3A14%3A58Z&se=2019-03-10T17%3A19%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**